impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Radar.js

```diff
@@ -165,6 +165,16 @@ define([
         var record = {
           // May be "" for the virtual null series.
           name: seriesEntity.label,
+
+          // For Radar, click event handler params do not contain the information about the
+          // data point that is clicked. It gives the entire series data.
+          // Because of which tooltip also shows the entire series data
+          // and Chart Interactions ( Select and Execute ) are using Series'
+          // visual key instead of a combination of series key and category key.
+          // Reference: https://github.com/apache/echarts/issues/16160 &
+          // https://github.com/apache/echarts/issues/10537#issuecomment-608339356
```

Comment:
I am assuming you weren't able to hack anything, even if not officially documented.

Reply:
Yes 😔